buku

```diff
@@ -1207,6 +1216,29 @@ def prompt(results, noninteractive=False):
             break
 
 
+def prompt_delete(results, noninteractive=False):
+    """Show each matching result from a search and prompt to delete"""
+
+    count = 0
+    for row in results:
+        count += 1
+        print_record(row, count)
+
+    if noninteractive:
```

Comment:
To delete blindly the search results is a dengerous approach.
I suggest to disable noninteractive mode for deletion,